frontend/src/pages/vpn-relay-welcome.page.tsx

```diff
@@ -85,7 +80,9 @@ const VpnRelayWelcome: NextPage = () => {
 
               <p>{l10n.getString("vpn-relay-go-vpn-body")}</p>
               <LinkButton
-                href={"https://relay.firefox.com/"}
+                href={`https://www.mozilla.org/en-US/products/vpn/download/?utm_source=${encodeURIComponent(
```

Comment:
I think if you leave out the language, it'll automatically redirect to the correct one:

```suggestion
                href={`https://www.mozilla.org/products/vpn/download/?utm_source=${encodeURIComponent(
```